webapp/public/locales/fr/translation.json

```diff
@@ -297,6 +297,13 @@
     "Note: You may need to change the file name to open the file on certain operating systems": "Remarque: vous devrez peut-être changer le nom du fichier pour ouvrir le fichier sur certains systèmes d'exploitation",
     "Next": "Continuer",
     "Previous": "Retourner",
-    "Application Status": "État de la candidature"
+    "User Title": "Titre de l'utilisateur",
+    "User Id": "Identifiant d'utilisateur",
```

Comment:
These two are missing in the english translation file, please add them there or remove them from here if they're not being used.